networks/rpc/http.go

```diff
@@ -48,9 +49,12 @@ var nullAddr, _ = net.ResolveTCPAddr("tcp", "127.0.0.1:0")
 
 type httpConn struct {
 	client    *http.Client
+	url       string
 	req       *http.Request
```

Comment:
```suggestion
```

Is this 'req' field needed? 
Ethereum also removed this field by adding the SetHeader method.
And all unit tests in the rpc directory pass even without this field.

Reply:
My mistake. Thank you for leting me know!